style.css

```diff
@@ -0,0 +1,126 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap");
+
+.container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.menu {
+  display: flex;
+  border: 8px;
+  margin: 10px;
+  padding: 8px;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+  justify-content: center;
+}
+
+.logo-placeholder {
+  display: flex;
+  padding: 10px 12px;
+  align-items: center;
+  justify-content: center;
+  border: 8px;
+  margin: 10px;
+}
+
+.my-logo {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.nav-button {
+  background: #fff;
+  border: none;
+  padding: 12px;
+  gap: 10px;
+}
+
+.nav-button:hover {
+  background: #6070ff;
+  box-shadow: 1px 8px 16px rgba(64, 83, 252, 0.24);
+}
+
+.nav-button:active {
+  background: #2230d2;
+}
+
+.headline {
+  display: flex;
+  align-items: center;
+  height: 480px;
+  padding: 114px 24px;
+  flex-wrap: wrap;
+}
+
```

Comment:
- I think it is important for you to follow the figma template as much as possible, because, from your project, I could see that there was no background image added to the `headline`  section of your portfolio 

| Figma Design  |Your Design |
| :---:   | :---: | 
| ![home](https://user-images.githubusercontent.com/99246359/198831542-1b908400-b1cd-4070-9c09-09855def74fc.PNG) |![prag](https://user-images.githubusercontent.com/99246359/198831553-be1ac94c-a1d6-4b7b-afcd-0de3dd252d29.PNG)  |  

- I think it will be better for you to fix the margin that I have highlighted with the blue arrow as well. 🤗